subprojects/scala/build.gradle.kts

```diff
@@ -29,8 +29,10 @@ dependencies {
     implementation(libs.guava)
     implementation(libs.inject)
 
-    compileOnly("org.scala-sbt:zinc_2.12:1.3.5") {
-        exclude(module="log4j-core") // Because not needed and vulnerable
+    compileOnly("org.scala-sbt:zinc_2.12:1.6.1") {
```

Comment:
Have we considered using the Scala 2.13 artifact instead?

Reply:
I haven't thought of that. I can update it for sure, though not sure if benefits will enormous, but I guess we would be fully up to date then.

Reply:
I think the main benefit would be avoiding the Scala 2.12 dependencies given that most projects should be using 2.13 by now.

Reply:
Changed to 2.13, let's see if everything passes.